crates/core_arch/src/loongarch32/mod.rs

```diff
@@ -0,0 +1,47 @@
+//! `LoongArch32` intrinsics
+
+use crate::arch::asm;
+
+#[allow(improper_ctypes)]
+unsafe extern "unadjusted" {
+    #[link_name = "llvm.loongarch.cacop.w"]
+    fn __cacop(a: i32, b: i32, c: i32);
+    #[link_name = "llvm.loongarch.csrrd.w"]
+    fn __csrrd(a: i32) -> i32;
```

Comment:
Would it make sense to have these in the common code returning `isize`? It is technically a breaking change but this is still an unstable feature.

Reply:
Thanks for your comments. I don't quite understand why this is considered a breaking change. Previously, LoongArch64 exported `csrrd(i32) -> i64`, and it sill does. Now, LoongArch32 is introducing `csrrd(i32) -> i32`, which doesn't change the existing behavior.

Reply:
I'm suggesting having a single shared intrinsic `csrrd(i32) -> isize` instead of separate ones for loongarch32 and loongarch64.